bundle/edu.gemini.p1monitor/src/main/scala/edu/gemini/p1monitor/P1MonitorMailer.scala

```diff
@@ -134,7 +134,7 @@ class P1MonitorMailer(cfg: P1MonitorConfig) {
 
   private def getInstrumentsString(prop: Proposal): String = prop.observations.map {
       obs => obs.blueprint match {
-        case Some(bp: GeminiBlueprintBase) => s"${bp.instrument.site.name} (${bp.instrument.id})"
```

Comment:
This is something we added on the 2016B to differentiate visitor instruments that can be in either site. In all honesty, we should likely remove the site in instrument, but that maybe involved, especially in terms of upconverting old proposals